RELEASES.md

```diff
@@ -0,0 +1,78 @@
+
+Maintenance releases will be supported for upto 6 months or more until the end of life is announced for the branch i.e. v1.9 will be EOL when v1.11 is released, as shown in the diagram.
+These branches will accept bug reports and backports until the end of life. All information will be populated below.
+
+The current state of available support will be:
+
+
+| Release                                                              | Status        | Start         | End of Life             |
+| ---------                                                            | ------------- |---------------|-------------------------|
+| 1.8                                                            | End of Life | Oct 22, 2020  | Jan 14, 2022            |
+| 1.9                                                            | Maintenance | Feb 10, 2022  | Until 1.11 is released. |
+| 1.10                                                            | Active | Sept 19, 2022 | 6 months or more        |
+| 1.11                                                            | Next | TBD           | TBD                     |
+
+
+## Kubernetes Support
+
+The Kubernetes matrix represents the versions of coredns that are used in the Kubernetes release. Kubernetes only supports `N-3` minor release versions, and it will ensure there is always a supported version of coredns available.
+
+| Kubernetes Version | Coredns Version |
+|--------------------|--------------------|
+| [1.24](https://github.com/kubernetes/kubernetes/blob/release-1.24/build/dependencies.yaml#L32)           | 1.8 (to be specific 1.8.6)    |
+| [1.25](https://github.com/kubernetes/kubernetes/blob/release-1.25/build/dependencies.yaml#L30)           | 1.9 (To be specific 1.9.3)    |
+| [1.26](https://github.com/kubernetes/kubernetes/blob/release-1.26/build/dependencies.yaml#L30)           | 1.9 (To be specific 1.9.3)    |
+| [1.27](https://github.com/kubernetes/kubernetes/blob/release-1.27/build/dependencies.yaml#L30)           | 1.10 (To be specific 1.10.1)   |
+
+
+## Backporting
+
+The maintainers are responsible for the backporting process. They will ensure any bug fixes that need to be in active release or any new features for the next major and minor releases.
+
+If there are important fixes that need to be backported, please let us know in one of three ways:
+- Open an issue.
+- Open a PR with a cherry-picked change from the master branch.
+- Open a PR with a ported fix.
+
+If there is no existing fix in the master, you should first fix it in the master by creating an issue. After that, once the PR is merged, open an issue on the release branch and backport it.
+If the issue is only in the release branch and not in the master, then open a PR and make the specific changes on the release branch to fix the issue.
+
+
+## Manual steps
+
+Manual steps for creating the major or minor release is present [here](https://github.com/coredns/coredns/blob/master/Makefile.release).
```

Comment:
I don't think this section needs to be in this document.

```suggestion
## Manual steps

Manual steps for creating the major or minor release is present [here](https://github.com/coredns/coredns/blob/master/Makefile.release).
```

Reply:
Done